module/VuFind/tests/unit-tests/src/VuFindTest/Mailer/MailerTest.php

```diff
@@ -452,23 +456,29 @@ public function testSendRecord()
         $driver = $this->createMock(\VuFind\RecordDriver\AbstractBase::class);
         $driver->expects($this->once())->method('getBreadcrumb')->willReturn('breadcrumb');
 
-        $viewCallback = function ($in) use ($driver): bool {
-            return $in['driver'] == $driver
-                && $in['to'] == 'to@example.com'
-                && $in['from'] == 'from@example.com'
-                && $in['message'] == 'message';
-        };
-        $view = $this->getMockBuilder(\Laminas\View\Renderer\PhpRenderer::class)
-            ->addMethods(['partial'])->getMock();
-        $view->expects($this->once())->method('partial')
-            ->with($this->equalTo('Email/record.phtml'), $this->callback($viewCallback))
-            ->willReturn('body');
+        $view = $this->createMock(\Laminas\View\Renderer\PhpRenderer::class);
+        $view->expects($this->once())->method('__call')
+            ->willReturnCallback(
+                function ($method, $args) use ($driver) {
+                    if ($method === 'partial') {
+                        $this->assertSame('Email/record.phtml', $args[0]);
+                        $in = $args[1];
+                        $this->assertSame($driver, $in['driver']);
+                        $this->assertSame('to@example.com', $in['to']);
+                        $this->assertSame('from@example.com', $in['from']);
+                        $this->assertSame('message', $in['message']);
+
+                        return 'body';
+                    }
+                    return null;
+                }
+            );
 
         $callback = function ($message): bool {
             return 'to@example.com' == $message->getTo()[0]->toString()
-                && 'from@example.com' == $message->getFrom()[0]->toString()
-                && 'body' == $message->getBody()->getBody()
-                && 'Library Catalog Record: breadcrumb' == $message->getSubject();
+            && 'from@example.com' == $message->getFrom()[0]->toString()
+            && 'body' == $message->getBody()->getBody()
+            && 'Library Catalog Record: breadcrumb' == $message->getSubject();
```

Comment:
Reverting more whitespace changes.

```suggestion
                && 'from@example.com' == $message->getFrom()[0]->toString()
                && 'body' == $message->getBody()->getBody()
                && 'Library Catalog Record: breadcrumb' == $message->getSubject();
```